independent-publisher-connectors/TSheets Quickbooks Time/apiDefinition.swagger.json

```diff
@@ -0,0 +1,654 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "QuickBooks Time",
+        "description": "Connector to read data from QuickBooks Time. QuickBooks Time formerly known as TSheets is a web-based and mobile time tracking and employee scheduling app. The service was accessed via a web browser or a mobile app. TSheets was an alternative to a paper timesheet or punch cards.",
+        "version": "1.0",
+        "contact": {
+            "name": "Artesian Software Technologies LLP",
+            "url": "https://artesian.io",
+            "email": "amit.sharma@artesian.io"
+        }
+    },
+    "host": "rest.tsheets.com",
+    "basePath": "/api/v1",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [],
+    "produces": [],
+    "paths": {
+        "/jobcodes": {
+            "get": {
+                "responses": {
+                    "default": {
```

Comment:
@artesianst default responses is not allowed. Please define a proper response, please see [here](https://learn.microsoft.com/en-gb/connectors/custom-connectors/coding-standards#responses)